crates/server/src/admin/storage/context.rs

```diff
@@ -29,12 +25,16 @@ pub fn delete_context(store: &mut Store, context_id: &str) -> eyre::Result<bool>
     }
 }
 
-pub fn get_context(store: &mut Store, context_id: &str) -> eyre::Result<Option<Context>> {
+pub fn get_context(store: &mut Store, context_id: &ContextId) -> eyre::Result<Option<Context>> {
     let did = get_or_create_did(store)?;
-    Ok(did.contexts.into_iter().find(|k| k.id == context_id))
+
+    todo!("Implement get_context")
+    // Ok(did.contexts.into_iter().find(|k| k.id == context_id))
 }
 
 pub fn get_contexts(store: &mut Store) -> eyre::Result<Vec<Context>> {
     let did = get_or_create_did(store)?;
-    Ok(did.contexts)
+
+    todo!("Implement get_contexts")
+    // Ok(did.contexts)
 }
```

Comment:
Moved `Context` out of `Did`, this pending logic depends on the unimplemented logic for reading the context metadata from the db that's beyond the scope of this patch.

Reply:
Do we have a gh issue to track this?

Reply:
Not needed, implementing it rn